timeless/reservations/views.py

```diff
@@ -92,45 +101,55 @@ def get(self, company_id):
         return jsonify(reservations_json)
 
 
-@bp.route("/")
-def list_reservations(reservations):
+@bp.route("/create", methods=("GET", "POST"))
+def create():
```

Comment:
@ebercamargo You need to use `CreateView` from `timeless/views.py`

Reply:
@vryazanov The list was created, according to the top of the file 'view.py', I will be set one :todo in the method, requesting another 30 minutes to create/delete...